README.md

```diff
@@ -152,15 +152,14 @@ If you'd like to get things running manually using Docker, you can do the follow
    docker run --rm -it --network=telemetry \
        --name frontend \
        -p 3000:8000 \
-       --read-only \
```

Comment:
why was this removed?

Reply:
This image writes still to `tmp`, and so the `--read-only` flag causes it to fail to start (since it's not allowed).

The "secure" version below starts a tmpfs up on `tmp` so that `--read-only` will continue to work, but it's more verbose and so it's nice to have the "simple" command here too.